src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -522,6 +469,82 @@ private void ApplyDecompressionSetting(HttpRequestMessage request, Decompression
                 _rtFilter.ClientCertificate = clientCertificates[0];
             }
         }
+
+        private RTPasswordCredential RTPasswordCredentialFromICredentials(ICredentials creds)
+        {
+            // The WinRT PasswordCredential object does not have a special credentials value for "default credentials".
+            // In general, the UWP HTTP platform automatically manages sending default credentials, if no explicit
+            // credential was specificed, based on if the app has EnterpriseAuthentication capability and if the endpoint
+            // is listed in an intranet zone.
+            //
+            // A WinRT PasswordCredential object that is either null or created with the default constructor (i.e. with
+            // empty values for username and password) indicates that there is no explicit credential. And that means
+            // that the default logged-on credentials might be sent to the endpoint.
+            //
+            // There is currently no WinRT API to turn off sending default credentials other than the capability
```

Comment:
Yes, in general setting a non-null credential will always prevent default credentials from being sent since there is an explicit credential.  However, having a "null" credential assigned to the properties doesn't mean "no credential".  It simply means "no explicit credential" and the end result can be either no credential is sent, or a default logged-on credentials is sent.